project.xml

```diff
@@ -41,6 +41,9 @@
         <option name = "use_checkout_explicit" value = "1" />
     </target>
 
+    <classfilename use-cxx = "true" source-extension = "cc" header-extension = "h"                                
```

Comment:
Line 15 in Travis section:
'option name = "dist" value = "trusty" '
'option name = "shadow_gcc" value = "2" '